### ISSUE TITLE
Day 3: Rucksack Reorganisation

### ISSUE THREAD

**Issue by rowlando** [rowlando/advent-of-code-2022-in-typescript#3]
https://adventofcode.com/2022/day/3

**Comment by rowlando** [comment]
This time I prompted Copilot with long meaningful function names to generate code. Occassionally I needed to help it along but all in all a very good job.

Here's a video of what it looked like:

[!(adventrucksack.webm)](https://github.com/rowlando/advent-of-code-2022-in-typescript/assets/89347/02f90592-8f35-471c-bd6d-ae44f712bf8b)

I'm using VSCode, with a build task set up watch the Typescript code and a launcher to run without debugging the currently selected file (the keyboard shortcut is ^F5).

I also use VSCode to refactor code by moving each function into its own file. 

To complete the code, I'll need to read the input file.
